bahmni-docker/README.md

```diff
@@ -272,12 +285,42 @@ By default, the configuration of openmrs and openmrsdb services are set to load
+|:----------------------|:-------------------------------------------------------------------------------------------|
+| PACS_DB_HOST          | The host name of the postgres server which runs database for dcm4chee and pacs-integration |
+| PACS_DB_PORT          | The port of the postgres server for dcm4chee and pacs-integration                          |
+| PACS_DB_ROOT_PASSWORD | The root password given to the postgres container ie pacsdb service                        |
+
+## DCM4CHEE Configurations
+| Variable Name        | Description                                                                                                                                                                           |
+|:---------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| DCM4CHEE_IMAGE_TAG   | This value specifies which image version needs to be used for reports service. List of tags can be found at [bahmni/dcm4chee - Tags](https://hub.docker.com/r/bahmni/dcm4chee/tags) . |
+| DCM4CHEE_DB_NAME     | The name of the database for dcm4chee service                                                                                                                                         |
+| DCM4CHEE_DB_USERNAME | Username for dcm4chee database                                                                                                                                                        |
+| DCM4CHEE_DB_PASSWORD | Password for dcm4chee database                                                                                                                                                        |
+
+## PACS-Integration Configurations
+| Variable Name                | Description                                                                                                                                                                                           |
+|:-----------------------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| PACS_INTEGRATION_IMAGE_TAG   | This value specifies which image version needs to be used for reports service. List of tags can be found at [bahmni/pacs-integration - Tags](https://hub.docker.com/r/bahmni/pacs-integration/tags) . |
+| PACS_INTEGRATION_DB_NAME     | The name of the database for pacs-integration service                                                                                                                                                 |
+| PACS_INTEGRATION_DB_USERNAME | Username for pacs-integration database                                                                                                                                                                |
```

Comment:
Username for pacs-intergation service to connect to pacs-integration postgres db.